### PR TITLE
ref(dashboards): decouple from ddm part 1

### PR DESCRIPTION
**Pull request by obostjancic**
- removes `convertToMetricWidget`
- adds utils for metric widgets in the dashboards
- removes metrics widgets from old widget viewer modal

### CODE REVIEW

File: static/app/components/modals/metricWidgetViewerModal.tsx [hunk 1]
```diff
@@ -25,20 +25,21 @@ import type {Organization} from 'sentry/types';
...
 import {OrganizationContext} from 'sentry/views/organizationContext';
 
-import {
-  convertToDashboardWidget,
-  convertToMetricWidget,
-  toMetricDisplayType,
-} from '../../utils/metrics/dashboard';
+import {convertToDashboardWidget} from '../../utils/metrics/dashboard';
 import {MetricWidget} from '../../views/ddm/widget';
 import {Tooltip} from '../tooltip';
```
Comment (ArthurKnaus) [inline static/app/components/modals/metricWidgetViewerModal.tsx:44]: nitpicking: not sure how, but we ended up with a few relative imports in a few places. 🤔  I find it rather hard to parse as soon as they utilize `../`. I will try to rewrite them whenever I encounter them.

**Review by ArthurKnaus — APPROVED** [review]
🥳

File: static/app/components/modals/metricWidgetViewerModal.tsx (hunk 1 above)
Comment (obostjancic) [reply static/app/components/modals/metricWidgetViewerModal.tsx:44]: I'll take care of these in a followup
